src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx

```diff
@@ -10,6 +10,7 @@ import LoadingIndicator from '@department-of-veterans-affairs/formation-react/Lo
 
 import { loadConnectedAccounts, deleteConnectedAccount } from '../actions';
 import { NoConnectedApps, ConnectedApps } from '../components';
+import { setPageFocus } from '../../../claims-status/utils/page';
```

Comment:
Is it kosher to import this from another application? If not, should I duplicate it here or  extract it to a higher level (maybe https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/accessibility/index.js)?

Reply:
I would much rather see this extracted to a common area than have them imported from another application. It's unclear to me how globally relevant the `#main h1` selector is. If it isn't nearly ubiquitous on the site then I might extract `setFocus` and duplicate `setPageFocus` in this app.

```
export function setFocus(selector) {
  const el =
    typeof selector === 'string' ? document.querySelector(selector) : selector;
  if (el) {
    el.setAttribute('tabIndex', -1);
    el.focus();
  }
}

export function setPageFocus(selector = '.va-nav-breadcrumbs') {
  const el = document.querySelector(selector);
  if (el) {
    setFocus(el);
  } else {
    setFocus('#main h1');
  }
}
```

It looks like there's already [a `focusElement` function](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/ui/index.js#L17) that looks similar to `setFocus`:

```
export function focusElement(selectorOrElement, options) {
  const el =
    typeof selectorOrElement === 'string'
      ? document.querySelector(selectorOrElement)
      : selectorOrElement;

  if (el) {
    if (el.tabIndex <= 0) {
      el.setAttribute('tabindex', '-1');
    }
    el.focus(options);
  }
}
```